### DISCUSSION TITLE
OWNPAD-etherpad outside of NextCloud

### DISCUSSION THREAD

**Discussion by scriptFactors** [otetard/ownpad#112] [Q&A]
hi -
tried with no success to access OWNPAD created etherpads directly from the etherpad instance.
i assume it is a COOKIE issue (msg is "You are not allowed to access this pad" or similar
studying the EP database i can not find any hint that would allow me to circumvent the obvious restriction - onlpy to see the OWNPAD created EPs have 4 cookies, instance created EPs only one...

any pointer most welcome!

salut -

**Comment by otetard** [comment]
In Ownpad, code related to the cookie is here:

https://github.com/otetard/ownpad/blob/77475b72fe6a0ff0a25c8174126069baaa85b12d/lib/Controller/DisplayController.php#L105

Basically, code is creating an Etherpad session (valid for one hour) and then creates a `sessionID` cookie for the right domain.
